### ISSUE TITLE
Add search for AUR packages by co-maintainer

### ISSUE THREAD

**Issue by alerque** [archlinuxfr/yaourt#367]
The `--maintainer` flag is great for getting a list of packages I maintain so that I can do automated things like check out all of the package builds at once. However I also co-maintain a number of packages and it would be nice to be able to search for these as well. A `--co-maintainer` flag would be a nice addition.

**Comment by alerque** [comment]
It looks like adding this feature might be dependent on the upstream RPC interface supporting such a query. I've [added a feature request](https://bugs.archlinux.org/task/57675) regarding that.

**Comment by f2404** [comment]
You're right, that would require support from the RPC interface which seems to be missing.
